container/Makefile

```diff
@@ -104,14 +104,11 @@ $(remote_containers):
 build_local_containers: $(local_containers)
 
 $(local_containers):
-ifneq ($(shell id -u), 0)
```

Comment:
This is to test if the user has sudo priviedges no?

Reply:
no longer a req w a recent apptainer